### PR TITLE
Reduce poll logging to debug

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Before this logged ~10 messages every few seconds and quite spamming the log.

- [x] read the [CONTRIBUTING guidelines](README.md#contributing)
- [ ] raised a GitHub issue or discussed it on the projects chat beforehand
- [ ] added unit tests
- [ ] added integration tests
- [ ] updated documentation if needed
- [ ] updated CHANGELOG.md

### CODE REVIEW

File: hscontrol/poll.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func logPollFunc(
 	node *types.Node,
 ) (func(string), func(error, string)) {
 	return func(msg string) {
-			log.Info().
+			log.Debug().
```
Comment (kradalby) [inline hscontrol/poll.go:35]: I think it in general could be made into Trace.

File: hscontrol/poll.go [hunk 2]
```diff
@@ -455,21 +455,21 @@ func (h *Headscale) handlePoll(
...
 				data, err = mapp.PeerRemovedResponse(mapRequest, node, update.Removed)
 			case types.StateSelfUpdate:
 				if len(update.ChangeNodes) == 1 {
-					logInfo("Sending SelfUpdate MapResponse")
+					logDebug("Sending SelfUpdate MapResponse")
 					node = update.ChangeNodes[0]
 					data, err = mapp.LiteMapResponse(mapRequest, node, h.ACLPolicy, types.SelfUpdateIdentifier)
 				} else {
-					logInfo("SelfUpdate contained too many nodes, this is likely a bug in the code, please report.")
+					logDebug("SelfUpdate contained too many nodes, this is likely a bug in the code, please report.")
```
Comment (kradalby) [inline hscontrol/poll.go:469]: This should probably be Warn instead.

File: hscontrol/poll.go [hunk 3]
```diff
@@ -30,9 +30,19 @@ type UpdateNode func()
 func logPollFunc(
 	mapRequest tailcfg.MapRequest,
 	node *types.Node,
-) (func(string), func(error, string)) {
+) (func(string), func(string), func(error, string)) {
 	return func(msg string) {
-			log.Info().
+			log.Trace().
```
Comment (SuperSandro2000) [inline hscontrol/poll.go:35]: I've never worked with zerolog before, so I am not sure if this is the way to go

Comment (kradalby) [reply hscontrol/poll.go:35]: Looks good 👍

**Review by kradalby — APPROVED** [review]

**Pull request merged**